src/config.js

```diff
@@ -51,6 +52,11 @@ const DefaultConfig = {
     },
     relay: {
       enabled: true,
+      advertise: {
+        bootDelay: RelayConstants.ADVERTISE_BOOT_DELAY,
+        enabled: true,
```

Comment:
This should be false by default since we disable HOP by default
```suggestion
        enabled: false,
```